__tests__/V4AWSClusterManagement.js

```diff
@@ -86,10 +86,10 @@ it('renders all the v4 AWS cluster data correctly without nodes ready', async ()
   expect(getAllByText(V4_CLUSTER.id)).toHaveLength(2);
   expect(
     getByTestId('desired-nodes').querySelector('div:nth-child(2)').textContent
-  ).toBe('3');
+  ).toBe('0');
   expect(
     getByTestId('running-nodes').querySelector('div:nth-child(2)').textContent
-  ).toBe('3');
+  ).toBe('0');
```

Comment:
why did this become 0?

Reply:
Same question I am asking to myself... Fixed for all v4 tests. Thanks!